.github/workflows/build.yml

```diff
@@ -65,7 +65,7 @@ jobs:
             runner: windows-2022
     runs-on: ${{ matrix.runner }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
```

Comment:
Can you add a comment with the version that the hash points to. Here it would be something like `# v3`. This would also applies to all other places where you have replaced the hash.